config/api-postman.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+return [
+
+    /**
+     * Structured
+     *
+     * If you want folders to be generated based on namespace.
+     */
+
+    'structured' => false,
+
+    /**
+     * Base URL
+     *
+     * The base URL for all of your endpoints.
+     */
+
+    'base_url' => 'https://api.example.com/',
+
+    /**
+     * Auth Middleware
+     *
+     * The middleware which wraps your authenticated API routes.
+     *
+     * E.g. auth:api, auth:sanctum
+     */
+
+    'auth_middleware' => 'auth:api',
```

Comment:
Can we change this to an array to allow multiples?

Reply:
Single auth detection, possibly multiple in the future.